fixed Subassembly .craft files and added descriptions.

speedwaystar opened the pull request:
i'm making progress on updating pricing and techtree positions based mostly off RP-0. in the meantime here are working versions of the Subassemblies, with descriptions.

speedwaystar committed:
fixed Subassembly .craft files and added descriptions.

raidernick commented:
@speedwaystar were these made with the latest release of the mod that uses firespitter to change the textures of a single set of parts? I removed all duplicates of the same part and combined them into 1 with texture change for each part. That is the reason I deleted the old crafts and subassemblies.

speedwaystar commented:
they were indeed. I've also set the appropriate paint schemes and block A/B
types for each craft.
- added craft descriptions
- fixed broken RN_Soviet_Rockets part names
- used Sputnik FSMeshSwitch for Sputnik, Vostok, Voskhod & Molniya launchers, and Soyuz FSMeshSwitch for Soyuz-FG.
- use correct(ish) FSTextureSwitch historical colour scheme for each launcher (block A & B)

On Mon, 18 Apr 2016, 9:00 p.m. raidernick, notifications@github.com wrote:

> @speedwaystar https://github.com/speedwaystar were these made with the
> latest release of the mod that uses firespitter to change the textures of a
> single set of parts? I removed all duplicates of the same part and combined
> them into 1 with texture change for each part. That is the reason I deleted
> the old crafts and subassemblies.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> https://github.com/KSP-RO/SovietRockets/pull/5#issuecomment-211370651

speedwaystar committed:
rebase Proton-K on remote master + description
